app.py

```diff
@@ -31,6 +35,10 @@ def create_app(config_name):
         )
     )
 
+    @app.route("/analytics", methods=['POST', 'GET'])
```

Comment:
Good work @Bonifase, I suggest you restrict the analytics report to Admin only

Reply:
Cool, let me put that into consideration.